system-modules.md

```diff
@@ -0,0 +1,15 @@
+# System Module
+
+## Get System Modules
+
+This method enables you to retrieve all system modules and returns a SystemModuleInterface interface.
```

Comment:
**suggestion(blocking):** Can we please have a link to the corresponding interface?